deployment/docker-compose.yml

```diff
@@ -56,63 +74,27 @@ web:
 
 # Exactly the same as web but for staging we use a different port.
 stagingweb:
-  image: nginx
-  hostname: nginx
-  volumes:
-    - ./sites-enabled:/etc/nginx/conf.d:ro
-    # I dont use volumes_from as I want to use the ro modifier
-    - ./static:/home/web/static:ro
-    - ./media:/home/web/media:ro
-    - ./logs:/var/log/nginx
-  links:
-    - uwsgi:uwsgi
+  <<: *web
   ports:
     - "61101:8080"
 
-
-# Local development docker deployment
+# Exactly the same as production but for dev env, we expose the port and uses
+# different port for the web.
 devdb:
-  image: kartoza/postgis
-  volumes:
-    - ./pg/postgres_data:/var/lib/postgresql
-  environment:
-    - USERNAME=docker
-    - PASS=docker
-  # RMN : Expose ports so we can access it from the host, maybe using pgAdmin
+  <<: *db
```

Comment:
Check this out @timlinux . I uses yml anchor and references to simplify it. Perhaps it is better to use extend keyword, but it is not really DRY at the moment.